agent/src/Lib/External/AppManifest.hs

```diff
@@ -49,55 +47,49 @@ instance FromJSON AssetMapping where
         assetMappingOverwrite <- o .: "overwrite"
         pure $ AssetMapping { .. }
 
-data AppManifest (n :: Nat) where
-    AppManifestV0 ::{ appManifestV0Id :: AppId
-                    , appManifestV0Version :: Version
-                    , appManifestV0Title :: Text
-                    , appManifestV0DescShort :: Text
-                    , appManifestV0DescLong :: Text
-                    , appManifestV0ReleaseNotes :: Text
-                    , appManifestV0PortMapping :: HM.HashMap Word16 Word16
-                    , appManifestV0ImageType :: ImageType
-                    , appManifestV0Mount :: FilePath
-                    , appManifestV0Assets :: [AssetMapping]
-                    , appManifestV0OnionVersion :: OnionVersion
-                    , appManifestV0Dependencies :: HM.HashMap AppId VersionRange
-                    } -> AppManifest 0
+data AppManifest where
+     AppManifest :: { appManifestId :: AppId
+                    , appManifestVersion :: Version
+                    , appManifestTitle :: Text
+                    , appManifestDescShort :: Text
+                    , appManifestDescLong :: Text
+                    , appManifestReleaseNotes :: Text
+                    , appManifestPortMapping :: HM.HashMap Word16 Word16
+                    , appManifestImageType :: ImageType
+                    , appManifestMount :: FilePath
+                    , appManifestAssets :: [AssetMapping]
+                    , appManifestOnionVersion :: OnionVersion
+                    , appManifestDependencies :: HM.HashMap AppId VersionRange
+                    } -> AppManifest
 
-hasUi :: forall n. AppManifest n -> Bool
-hasUi AppManifestV0 {..} = isJust $ HM.lookup 80 appManifestV0PortMapping
+hasUi :: AppManifest -> Bool
+hasUi AppManifest {..} = isJust $ HM.lookup 80 appManifestPortMapping
```

Comment:
this is redundant with `uiAvailable` delete one of them, IDC which one, they seem equivalent both in perf and semantics